client/src/components/board.js

```diff
@@ -52,11 +52,29 @@ class Board extends React.Component {
   }
 
   render() {
+    const copyToClipboard = function copyToClipboard() {
+      document.getElementById('urlInput').value = window.location.href;
```

Comment:
I really don't like the idea of using the document location here. We should utilize the UID of the board to generate the proper link that can be copied. I also don't want to tackle it this way unless we can give some visual feedback *and* display the url to the user at the same time. I kinda want to stick to a basic "show the link and let them copy it" approach until we get into polish